deps/Makefile

```diff
@@ -1709,6 +1709,10 @@ $(SRCDIR)/srccache/libunwind-$(UNWIND_VER)/configure: $(SRCDIR)/srccache/libunwi
 	$(JLCHECKSUM) $<
 	cd $(dir $<) && $(TAR) xfz $<
 	cd $(dir $<)/libunwind-$(UNWIND_VER) && patch -p1 < $(SRCDIR)/libunwind.patch
+	cd $(dir $<)/libunwind-$(UNWIND_VER) && patch -p1 < $(SRCDIR)/libunwind-001-invalid-dwarf-opcodes-can-cause-references-beyond.patch
+ifneq (,$(findstring aarch64,$(ARCH)))
+	cd $(dir $<)/libunwind-$(UNWIND_VER) && patch -p1 < $(SRCDIR)/libunwind-aarch64.patch && autoreconf -fi
+endif
```

Comment:
historical commentary: this folder is named `deps` because it is excluded from language stats for the repo. (https://github.com/JuliaLang/julia/issues/726)